pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -96,9 +96,10 @@ type Image struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	Repo   string   `json:"repo,omitempty"`
-	Digest string   `json:"digest,omitempty"`
-	Tags   []string `json:"tags,omitempty"`
+	Project string   `json:"project,omitempty"`
```

Comment:
This is probably a no-go. I don't think any other types expose Project as a field like this.